packages/app/.eslintrc.js

```diff
@@ -11,11 +11,12 @@ module.exports = {
   plugins: ['ember'],
   extends: ['eslint:recommended', 'plugin:ember/recommended', 'prettier'],
   env: {
-    browser: true
+    browser: true,
+    es6: true
   },
   rules: {
     'multiline-comment-style': ['error', 'starred-block'],
-    'ember/no-jquery': 'error'
+    'ember/no-jquery': 'off'
```

Comment:
Do we use jQuery in the app or only in tests?

Reply:
We use it in one test

Reply:
If so, can we turn this off only in the test dir?